### PR TITLE
Preso-ro presentation generator

### PR DESCRIPTION
**Pull request by mathewdbutton**
Hey Lads,

So here is the idea I was chatting to you both about. This is the cleaned up version of what I used to generate this months presentation.

It's aim is to try and automate the easy bits that are pretty consistent each month while still allowing you to add notes and move things around. It's still pretty rough, but I've finally got in into a form where you can kind of understand what it's doing.

Please don't be gentle, it may not be useful, and that's fine, as I've had quite a bit of fun writing this regardless of the outcome, so happy if you don't think it'll be of any use.

Anyhoo, I've written a README.md to describe how to use it, and if you find the time, I'd love it if you could have a look at some stage.

### CODE REVIEW

File: preso-ro/preso_ro.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+require_relative "./poros/sponsor"
+require_relative "./poros/speaker"
+require_relative "./poros/presentation"
+require_relative "./poros/company"
+
+class PresoGo
+
+  def self.generate(path, pres_prefix)
```
Comment (mathewdbutton) [inline preso-ro/preso_ro.rb]: This could certainly do with some refactoring, so just squint if you get this far.

File: preso-ro/speakers.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+-
+  :name: "Sameera Gayan"
```
Comment (mathewdbutton) [inline preso-ro/speakers.yml]: This part might seem a little labourious but it would help out with repeat speakers.

**Comment by paulfioravanti** [comment]
Hey @mathewdbutton, so I still haven't gotten around to reviewing this yet (sorry!), but I was thinking that perhaps it might be worth putting this application in a completely new repo under the rails-oceania organisation, rather than in here, since other ROROs could potentially contribute to and use it. So, unless you want to keep the app under your own Github account, you should have the correct permissions to create a new one under rails-oceania. What do you think?

**Comment by mathewdbutton** [comment]
Hey @paulfioravanti 

I had thought of doing this, I just not convinced it's actually useful yet and was wanting to trial it out a bit more with ROROSyd before going "HERE'S THE THING". As it's
1) pretty hard to read at the moment, it's basically a stored procedure, with some glitter
2) It's very opinionated as to how one should do things

So yes, I would love to share it around but want to get it in a state where I think it's actually helpful and it's not ugly as sin.

> Hey @mathewdbutton, so I still haven't gotten around to reviewing this yet (sorry!)

It's really no problem, I don't expect you to do it at all, just wanted to share what I was working on and get some feedback is all.

**Pull request merged**